### PR TITLE
Add links to metabase dashboards as admin

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: app/controllers/concerns/sortable.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def sortable_columns
   end
 
   def order_by
-    "#{sort_column} #{sort_direction}"
+    "#{sort_column} #{sort_direction} NULLS LAST"
```
Comment (hopsoft) [inline app/controllers/concerns/sortable.rb]: This looks like a possible SQL injection vulnerability. We should probably wrap this with some defensive checks.

File: app/views/advertiser_dashboards/show.html.erb [hunk 2]
```diff
@@ -14,6 +14,10 @@
 
 <main id="content" role="main" class="bg-light">
   <div class="container space-1">
+    
+    <% if authorized_user.can_admin_system? || current_user != true_user %>
+      <small class="text-center d-block pb-3"><%= link_to "open metabase dashboard in new window", "https://metabase.codefund.io/dashboard/10?date_filter=thismonth&advertiser_user_id=#{current_user.id}", class: "text-link", target: "_blank" %></small>
+    <% end %>
```
Comment (hopsoft) [inline app/views/advertiser_dashboards/show.html.erb]: Is `current_user != true_user` too permissive here?

File: app/views/publisher_dashboards/show.html.erb [hunk 3]
```diff
@@ -14,6 +14,10 @@
 
 <main id="content" role="main" class="bg-light">
   <div class="container space-1">
+    
+    <% if authorized_user.can_admin_system? || current_user != true_user %>
```
Comment (hopsoft) [inline app/views/publisher_dashboards/show.html.erb]: Same authorization related question as above.

File: app/views/advertiser_dashboards/show.html.erb (hunk 2 above)
Comment (coderberry) [reply app/views/advertiser_dashboards/show.html.erb]: Only admins can masquerade right? Is there a better way?

File: app/controllers/concerns/sortable.rb (hunk 1 above)
Comment (coderberry) [reply app/controllers/concerns/sortable.rb]: The sort column and direction are being restricted to certain fields:

```ruby
  def order_by
    "#{sort_column} #{sort_direction} NULLS LAST"
  end

  def sort_column
    return params[:column] if sortable_columns.include?(params[:column])
    return "name"          if sortable_columns.include?("name")
    "created_at"
  end

  def sort_direction
    %w[asc desc].include?(params[:direction]) ? params[:direction] : "asc"
  end
```

Comment (hopsoft) [reply app/controllers/concerns/sortable.rb]: Nice job. I should have done more research.

File: app/views/advertiser_dashboards/show.html.erb (hunk 2 above)
Comment (hopsoft) [reply app/views/advertiser_dashboards/show.html.erb]: Yes. This might be a better solution. 

```
AuthorizedUser.new(true_user).can_admin_system?
```

**Pull request merged**